librz/core/cdebug.c

```diff
@@ -642,3 +642,41 @@ RZ_API void rz_debug_traces_ascii(RzDebug *dbg, ut64 offset) {
 	rz_table_free(table);
 	rz_list_free(info_list);
 }
+
+/**
+ * \brief Close debug process (Kill debugee and all child processes)
+ * \param core The RzCore instance
+ * \return success
+ */
+RZ_API bool rz_core_debug_process_close(RzCore *core) {
+	rz_return_val_if_fail(core && core->dbg, false);
+	RzDebug *dbg = core->dbg;
+	// Stop trace session
+	if (dbg->session) {
+		rz_debug_session_free(dbg->session);
+		dbg->session = NULL;
+	}
+#ifndef SIGKILL
+#define SIGKILL 9
+#endif
```

Comment:
why this? I think SIGKILL should be defined by the system, not by us.

Reply:
right, so i think there should be no IFNDEF, no? on the systems where it's not define it's probably safe to not use it and not even compile the code using it.

Reply:
I think it's handled by the `rz_debug_kill()` function with the corresponding backends.

Reply:
See, for example `librz/debug/p/native/windows/windows_debug.c` function `w32_kill()` - it simply ignored the value of the `sig`.

Reply:
I think rz_debug_kill(..., SIGKILL) should not even be called/executed/compiled if SIGKILL itself is not defined. But it's fine for now.